From: Adam Williamson <awilliam@redhat.com>
Date: Sat, 25 Mar 2023 09:19:41 -0700
Subject: screenshot: Fix broken GLib.Error.matches call

You have to pass a domain and an error code, not just an error
code.

Fixes https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/6540

Signed-off-by: Adam Williamson <awilliam@redhat.com>
Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2725>

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/a093a72
---
 js/ui/screenshot.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/screenshot.js b/js/ui/screenshot.js
index ecbe50b..d25744e 100644
--- a/js/ui/screenshot.js
+++ b/js/ui/screenshot.js
@@ -2084,7 +2084,7 @@ function _storeScreenshot(bytes, pixbuf) {
         try {
             bookmarks.load_from_file(recentFile);
         } catch (e) {
-            if (!e.matches(GLib.BookmarkFileError.FILE_NOT_FOUND)) {
+            if (!e.matches(GLib.BookmarkFileError, GLib.BookmarkFileError.FILE_NOT_FOUND)) {
                 log(`Could not open recent file ${uri}: ${e.message}`);
                 return;
             }
